### PR TITLE
Fix for misnamed Joomla3 module in docs

### PR DESCRIPTION
**Pull request by GreatEmerald**
Fixes issue #270

### CODE REVIEW

**Comment by pathawks** [comment]
Looks like we need to drop Ruby 1.9 from the test suite.

**Comment by GreatEmerald** [comment]
Added another small improvement: the category field is numerical, not a string.

File: site/_importers/joomla3.md [hunk 1]
```diff
@@ -11,18 +11,18 @@ To import your posts from a [Joomla 3](http://joomla.org) installation, run:
...
+      "category" => category,
       "prefix"   => "mytableprefix"
     })'
 {% endhighlight %}
 
 The only required fields are `dbname`, `prefix` and `user`. `password` defaults to `""`,
 and `host` defaults to `"localhost"`.
 
-If the `category` field is not filled, all articles will be imported, except the ones that are 
+If the `category` numerical field is not filled, all articles will be imported, except the ones that are
```
Comment (parkr) [inline site/_importers/joomla3.md]: Is the category *always* numerical?

**Comment by parkr** [comment]
Thanks! One question above.

**Comment by GreatEmerald** [comment]
In the current implementation of the importer, yes. Joomla stores categories as numbers; there is also a table that maps those numbers into human-readable names. It would be nice to make use of that, but it would make the tool quite a bit more complex.

**Comment by parkr** [comment]
@jekyllbot: merge +site

**Pull request merged**